pages/events/[id]/index.js

```diff
@@ -83,6 +93,23 @@ export default function EventDetails() {
               <p>Tasks: {event.tasks}</p>
               <p>Ideas, Message, Thoughts etc: {event.ideas}</p>
               <p>Guests: {event.guests}</p>
+              <h3>EXPENSES</h3>
+              <h4>BUDGET ${event.budget}</h4>
+              <p>Food & Drinks ${event.food}</p>
+              <p>Accomodation ${event.accomodation}</p>
+              <p>Transport ${event.transport}</p>
+              <p>Gifts ${event.gifts}</p>
+              <p>Other Expenses ${event.otherExpenses}</p>
```

Comment:
This is a good opportunity to use a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) to be more accessible.